### ISSUE TITLE
Larva testtool cannot access PostgreSQL XA datasource

### ISSUE THREAD

**Issue by gvanbrakel** [ibissource/iaf#1086]
**Describe the bug**
The framework currently supports PostgreSQL databases. Without XA, all tests of iaf-test pass. When XA is enabled, however,
quite a number of test scenarios fail with ``(ClassCastException) org.postgresql.xa.PGXADataSource cannot be cast to javax.sql.DataSource``

**Reporter**
Gerrit
**Category** 
Use following Tags to categorize your bug report
 - Larva (unit test tool)

**To Reproduce**
1. Checkout the latest version of the iaf-project
2. in test/src/main/resources/Test.properties, set ``jdbc.dbms.default=postgres-xa``
3. enable XA, e.g. using -Dapplication.server.type.custom=ACTIVEMQ
4. Start the postgres docker container via test/src/main/toos/setupDB/Postgresql/start.bat
5. Start the ibis
6. See that queries like `` select * from ibisprop `` can be executed via the console 'ExecuteJdbcQuery'
7. Start the larva test, see that the first already fails with ``(ClassCastException) org.postgresql.xa.PGXADataSource cannot be cast to javax.sql.DataSource``

**Expected behavior**
I expect that larva can access the same datasources as the framework itself.

**Screenshots**

**Environment:**
 - IAF Version 7.6 after PR #1083 

**Additional context**
```
java.lang.ClassCastException: org.postgresql.xa.PGXADataSource cannot be cast to javax.sql.DataSource
	at nl.nn.adapterframework.jdbc.JdbcFacade.getDatasource(JdbcFacade.java:130)
	at nl.nn.adapterframework.jdbc.JdbcFacade.getConnection(JdbcFacade.java:244)
	at nl.nn.adapterframework.jdbc.JdbcSenderBase.open(JdbcSenderBase.java:82)
	at nl.nn.adapterframework.jdbc.JdbcQuerySenderBase.open(JdbcQuerySenderBase.java:181)
	at nl.nn.adapterframework.jdbc.JdbcQuerySenderBase$$FastClassBySpringCGLIB$$7ac72a81.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at nl.nn.adapterframework.jdbc.FixedQuerySender$$EnhancerBySpringCGLIB$$2f4818f7.open(<generated>)
	at nl.nn.adapterframework.testtool.TestTool.openQueues(TestTool.java:1512)
	at nl.nn.adapterframework.testtool.TestTool.runScenarios(TestTool.java:285)
	at nl.nn.adapterframework.testtool.TestTool.runScenarios(TestTool.java:148)
	at nl.nn.adapterframework.testtool.TestTool.runScenarios(TestTool.java:134)
	at org.apache.jsp.larva.index_jsp._jspService(index_jsp.java:101)
```

**Comment by jacodg** [comment]
jdbc.dbms.defaults in "steps to reproduce" must be jdbc.dbms.default

**Comment by gvanbrakel** [comment]
N.B.:
jdbc.dbms.default=postgres-xa werkt i.c.m. BTM, **niet** zonder BTM
jdbc.dbms.default=postgres werkt **niet** i.c.m. BTM, **wel** zonder BTM

**Comment by gvanbrakel** [comment]
I could test locally with this hacky-hacky workaround:
```
if (datasourceName.endsWith("-xa")) {
	datasourceName = datasourceName.substring(0, datasourceName.length()-3);
}
```
added at three locations just before where setDatasourceName() was called

**Comment by jacodg** [comment]
Should be fixed with #1157
